desafio-8/desafio-8.css

```diff
@@ -0,0 +1,78 @@
+header {
+ padding: 20px 0;
+ background-color: #9932CC;
+ color: #ffffff;
+ display: flex;
+ align-items: center;
+ justify-content: center;
+ flex-direction: row;
+}
+
+body{
+ background-color:#222222;
+ font-family: Inter;
+}
+
+main {
+ width: 940px;
```

Comment:
this it's not bad to mobile?  you define a fixed width but the phone doesn’t  have this width

Reply:
yes, it doesn't make sense on mobile, I removed and created a mobile media query. What do you think?